openlibrary/templates/type/edition/render_summary.html

```diff
@@ -0,0 +1,21 @@
+$def with (page,work,edition,ocaid,work_title)
```

Comment:
```suggestion
$def with (work, edition, ocaid, work_title, star_ratings_stats)
```
`page` isn't used in this template, so let's remove it.

I was checking our render times, and it can take StarRatingsStats more time to render than many other components on the book page.  It's probably best to render it once, and pass it to this template.